lib/assessment.md

```diff
@@ -138,12 +157,12 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+div and divOne would appear on the same line, with divo to the left and divOne to the right. divFour would appear fully below .divFour.
```

Comment:
No, this isn't correct. The end result will looks something like this:
```
[2] [3]            [1]
[4] [5]
```

It may be worthwhile for you to drop this HTML and CSS into Codepen and play around with it - that might help you build up an intuition about `float` and `clear`.